From: =?utf-8?q?Christian_G=C3=B6ttsche?= <cgzones@googlemail.com>
Date: Fri, 14 Aug 2020 12:50:51 +0200
Subject: doveadm-director.1: drop acute accent

Found by Lintian:

    This manual page uses the \' groff sequence. Usually, the intent to
    generate an apostrophe, but that sequence actually renders as a an acute
    accent.

    For an apostrophe or a single closing quote, use plain '. For single
    opening quote, i.e. a straight downward line ' like the one used in
    shell commands, use &#92;(aq.
---
 doc/man/doveadm-director.1.in | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/doc/man/doveadm-director.1.in b/doc/man/doveadm-director.1.in
index c430a3d..069241e 100644
--- a/doc/man/doveadm-director.1.in
+++ b/doc/man/doveadm-director.1.in
@@ -108,7 +108,7 @@ hosts. All the existing connections will be kicked. If
 is specified, a flush script is also automatically executed.
 .PP
 Because the kicking and moving of users to new backends creates a temporary
-load spike, all the users aren\'t moved at once. The
+load spike, all the users aren't moved at once. The
 .B \-\-max\-parallel
 parameter specifies how many users can be moved concurrently.
 The default is 100.
@@ -116,7 +116,7 @@ The default is 100.
 If
 .B \-f
 parameter is used, the user associations are simply dropped. Existing
-connections won\'t be kicked and flush scripts aren\'t run.
+connections won't be kicked and flush scripts aren't run.
 .\"-------------------------------------
 .SS director kick
 .B doveadm director kick
@@ -280,4 +280,4 @@ user would be redirected to 192.168.10.3.
 @INCLUDE:reporting-bugs@
 .\"------------------------------------------------------------------------
 .SH SEE ALSO
-.BR doveadm (1)
\ No newline at end of file
+.BR doveadm (1)
